Motion Amendments 2019 - Vetting procedure (#28)
thewca/wca-documents: documents/motions/10.2017.0 - Committees and Teams.md

2. All WCA Committees and Teams shall be formed for an indefinite period, unless otherwise decided by the WCA Board of Directors.
3. Each WCA Committee and Team will have a direct relationship with at least one Director for management, communication, and escalation.
4. Each WCA Committee and Team shall have one Leader.
   1. The Leader of every WCA Committee and Team is chosen by the WCA Board of Directors, based on a voting in the WCA Board of Directors with at least supermajority, after a request for applications among the Community.

>       1. Senior Delegates and sitting Committee Leaders must be provided a one week time period to question, interview and debate the proposed Committee Leader.

Reviewer: ```suggestion
      1. Sitting Senior Delegates and Committee/Team Leaders must be provided with at least a one week time period to question, interview, and debate the proposed candidate.
```

>       1. Senior Delegates and sitting Committee Leaders must be provided a one week time period to question, interview and debate the proposed Committee Leader.

Reviewer: ```suggestion
      1. Sitting Senior Delegates and Committee/Team Leaders must be provided with at least a one week time period to question, interview, and debate with the proposed candidate.
```

>       2. The choice of the new Committee Leader must be confirmed in a vote among the Senior Delegates and all other sitting Committee Leaders with a majority, after the debate period has ended.

Reviewer: ```suggestion
      2. The choice of the new Committee/Team Leader must be confirmed in a vote among all the sitting Senior Delegates and Committee/Team Leaders with a majority, after the debate period has ended.
```

>       2. The choice of the new Committee Leader must be confirmed in a vote among the Senior Delegates and all other sitting Committee Leaders with a majority, after the debate period has ended.

Reviewer: ```suggestion
      2. The choice of the new Committee/Team Leader must be confirmed in a vote among all the sitting Senior Delegates and Committee/Team Leaders with a majority, after the debate period has ended.
```

>       3. After confirmation, the Committee Leader will be appointed by the WCA Board of Directors.

Reviewer: ```suggestion
      3. After confirmation, the Committee/Team Leader will be appointed by the WCA Board of Directors.
```

>       3. After confirmation, the Committee Leader will be appointed by the WCA Board of Directors.

Reviewer: ```suggestion
      3. After confirmation, the Committee/Team Leader will be appointed by the WCA Board of Directors.
```

>       4. In case there is no majority, another Leader must be proposed by the WCA Board of Directors.

Reviewer: ```suggestion
      4. In case there is no majority to confirm the selection, another candidate must be proposed by the WCA Board of Directors.
```
